airflow/models/trigger.py

```diff
@@ -90,8 +90,17 @@ def __init__(
     @internal_api_call
     def from_object(cls, trigger: BaseTrigger) -> Trigger:
         """Alternative constructor that creates a trigger row based directly off of a Trigger object."""
+        from airflow.models.crypto import get_fernet
```

Comment:
Testing conversation resolution

Reply:
You can try to dismiss your approval first; I think the conversations opened by a reviewer who approved the PR will be ignored.

Reply:
Just tested, it doesn't work 😞

Reply:
Works now after https://github.com/apache/airflow/pull/36496